spec/unit/provider/package/puppet_gem_spec.rb

```diff
@@ -67,6 +67,14 @@
       expect(described_class).to receive(:execute_gem_command).with(provider_gem_cmd, %w{uninstall --executables --all myresource --force --bindir=/usr/bin}).and_return('')
       provider.uninstall
     end
+
+    it 'should invalidate the rubygems cache' do
+      @gem_source = double('gem_source')
+      allow(Puppet::Util::Autoload).to receive(:gem_source).and_return(@gem_source)
+      expect(described_class).to receive(:execute_gem_command).with(provider_gem_cmd, %w{uninstall --executables --all myresource}).and_return('')
```

Comment:
Not sure this expect is needed as it is alos tested on [:61](https://github.com/alexjfisher/puppet/blob/PUP-10319/spec/unit/provider/package/puppet_gem_spec.rb#L61)

Reply:
Line removed.

Reply:
Without this line, builds were failing with `Provider puppet_gem package command 'gem' does not exist on this host`

Reply:
Yes, sorry i didn't see that this line is also returning something
It is ok like this 👍